README.md

```diff
@@ -21,36 +25,53 @@ var sharetools = new ShareTools({
             subject: 'SUPER IMPORTANT EMAIL',
             message: 'BBC News has new bespoke'
         },
-        app: {
-            title: 'Shiny new app',
-            text: 'Hello world'
+        // you can also specify custom social networks, e.g.
+        whatsapp123: {
```

Comment:
Hmm, I'm not sure that's the way forward - networks are networks, custom or not, we just arbitrarily went for FB/Twitter/Email.
If anything, we need to make our instantiation of _all_ networks more consistent, i.e. instead of:

```
        email: {
            subject: 'SUPER IMPORTANT EMAIL',
            message: 'BBC News has new bespoke'
        },
        // you can also specify custom social networks, e.g.
        whatsapp123: {
            shareEndpoint: 'http://example.com',
            popup:         true,
            properties: {
                name: 'WhatsApp',
                age:  '23',
                subject: 'Testing custom networks'
            }
        }
```

It should be:

```
        email: {
            shareEndpoint: ':mailto',
            popup: false,
            properties: {
                subject: 'SUPER IMPORTANT EMAIL',
                message: 'BBC News has new bespoke'
            }
        },
        // you can also specify custom social networks, e.g.
        whatsapp123: {
            shareEndpoint: 'http://example.com',
            popup:         true,
            properties: {
                name: 'WhatsApp',
                age:  '23',
                subject: 'Testing custom networks'
            }
        }
```

I'll raise an issue for this I think.